pom.xml

```diff
@@ -99,6 +99,11 @@
             <version>${java.plugin.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
This dependency is already declared, but with a test scope. I guess we now need it with compile (or even better, provided) scope. Let's remove this block and add provided or compile to the other one. I had to do the same for the other PR.